examples/tokenvm/cmd/token-cli/cmd/action.go

```
@@ -798,14 +793,8 @@ var exportAssetCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		if err := submit(ctx); err != nil {
-			return err
-		}
-		success, err := cli.WaitForTransaction(ctx, tx.ID())
-		if err != nil {
-			return err
-		}
-		printStatus(tx.ID(), success)
+
+		_ = waitForTransactionAndPrintStatus(ctx, cli, submit, tx)
```

You should not ignore this error